### PR TITLE
feat(tests): add first test to ConnectionFromMongoCursor

### PR DESCRIPTION
**Pull request by JCMais**
Also added commitlint and migrated hooks to husky

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "presets": ["flow", "es2015", "stage-0"]
+  "plugins": [
+    ["transform-flow-strip-types"]
```
Comment (sibelius) [inline .babelrc]: why not flow preset?

File: .babelrc [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+  ],
+  "presets": ["es2015", "stage-0"]
```
Comment (sibelius) [inline .babelrc]: we should embrave preset env

File: .flowconfig [hunk 3]
```diff
@@ -9,3 +9,13 @@ node_modules/graphql-relay/lib/index.js.flow
...
+
+suppress_type=$FlowIssue
+suppress_type=$FlowFixMe
+suppress_type=$FlowFixMeProps
+suppress_type=$FlowFixMeState
+
+suppress_comment=\\(.\\|\n\\)*\\$FlowFixMe\\($\\|[^(]\\|(\\(<VERSION>\\)? *\\(site=[a-z,_]*[react_native_oss|react_native_fb][a-z,_]*\\)?)\\)
+suppress_comment=\\(.\\|\n\\)*\\$FlowIssue\\((\\(<VERSION>\\)? *\\(site=[a-z,_]*[react_native_oss|react_native_fb][a-z,_]*\\)?)\\)?:? #[0-9]+
+suppress_comment=\\(.\\|\n\\)*\\$FlowFixedInNextDeploy
+suppress_comment=\\(.\\|\n\\)*\\$FlowExpectedError
```
Comment (sibelius) [inline .flowconfig]: nice

File: commitlint.config.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        'fix',
+        'perf',
+        'refactor',
+        'revert',
+        'style',
+        'test',
+      ],
+    ],
+  },
+};
```
Comment (sibelius) [inline commitlint.config.js]: nice

File: src/__tests__/ConnectionFromMongoCursor.spec.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  const GroupB = await createGroup();
+
+  await createUser({ group: GroupA });
+  await createUser({ group: GroupB });
+
+  const cursor = UserModel.find();
+
+  const context = { randomProperty: 2 };
+  const loader = jest.fn();
+  loader.mockReturnValue('user');
```
Comment (sibelius) [inline src/__tests__/ConnectionFromMongoCursor.spec.js]: great

File: .circleci/config.yml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+      - run:
+          name: Run lint
+          command: yarn lint
+      - run:
+          name: Run tests
+          command: yarn test
+      - save_cache:
+          key: v0-mongodb-{{ .Branch }}
+          paths:
+            - $HOME/.mongodb-binaries
```
Comment (jgcmarins) [inline .circleci/config.yml]: nice one

**Pull request merged**